edge/pkg/devicetwin/devicetwin_test.go

```diff
@@ -94,7 +98,8 @@ func TestStart(t *testing.T) {
 	ormerMock.EXPECT().QueryTable(gomock.Any()).Return(querySeterMock).Times(1)
 	querySeterMock.EXPECT().All(gomock.Any()).Return(int64(1), nil).Times(1)
 	go dt.Start()
-	time.Sleep(1 * time.Millisecond)
+	time.Sleep(delay)
+	retry++
```

Comment:
Need `retry++` here?

Reply:
it starts from 0,  the sleep here counts, so it will consistent with other occurrence (5 times total).

Reply:
ok, sounds fine :)